modules/tournament/src/main/TournamentScheduler.scala

```diff
@@ -404,41 +404,38 @@ Thank you all, you rock!"""
           // Because berserking lowers the player rating
           _ map { _.copy(noBerserk = true) }
         },
-      // hourly crazyhouse tournaments!
+      // hourly crazyhouse/chess960 tournaments!
```

Comment:
doesn't that reduce the amount of ZH hourly tournaments? I don't mind it, but maybe some will.

Reply:
Yeah, it's basically a trade off; more 960, anti, and atomic, but also fewer hourly ZH tournaments. Considering these 4 variants are about equal in popularity, I figured this was the best solution as they all have the same amount of tournaments this way.